### PR TITLE
Add CommonJS support back

### PR DESCRIPTION
**Pull request by CahidArda**
CommonJS was removed in https://github.com/upstash/qstash-js/pull/166/files

Added CommonJS back and reverted path changes in package.json.

**Should be merged after testing with a canary release.**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,14 +106,14 @@
...
     "typescript": "^5.4.5",
     "undici-types": "^6.16.0",
-    "@solidjs/start": "^1.0.6",
-    "@sveltejs/kit": "^2.5.18",
-    "hono": "^4.5.8",
-    "h3": "^1.12.0",
     "vitest": "latest"
   },
   "dependencies": {
+    "@solidjs/start": "^1.0.6",
```
Comment (ytkimirti) [inline package.json:112]: We should not put these into dependencies, or devDependencies. They should be in optionalDependencies instead

File: package.json [hunk 2]
```diff
@@ -12,51 +12,66 @@
   "bugs": {
     "url": "https://github.com/upstash/sdk-qstash-ts/issues"
   },
+  "main": "./index.js",
+  "module": "./index.mjs",
+  "types": "./index.d.ts",
   "files": [
-    "dist"
+    "./**"
```
Comment (ytkimirti) [inline package.json:19]: This should stay as dist

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -24,11 +24,6 @@ jobs:
       - name: Set package version
         run: echo $(jq --arg v "${{ env.VERSION }}" '(.version) = $v' package.json) > package.json
 
-      - name: Setup Bun
-        uses: oven-sh/setup-bun@v1
-        with:
```
Comment (ytkimirti) [inline .github/workflows/release.yml:29]: Does it work without this?

**Comment by ytkimirti** [comment]
And there is a typo in the dependencies. We should probably remove that whitespace @CahidArda 
```
  "dependencies": {
    "crypto-js": ">=4.2.0",
    "jose": "^ 5.2.3"
  }
  ```

File: .github/workflows/release.yml (hunk 3 above)
Comment (CahidArda) [reply .github/workflows/release.yml:29]: it does 

https://github.com/upstash/qstash-js/actions/runs/10771948555https://www.npmjs.com/package/@upstash/qstash/v/2.7.6-canary

**Pull request merged**